### PR TITLE
GPG in integration test

### PR DESCRIPTION
**Pull request by tummychow**
vagrant integration tests should pass on this branch.

### CODE REVIEW

File: integration/single-node-slug-deploy/check.go [hunk 1]
```diff
@@ -62,6 +163,29 @@ func main() {
 	}
 }
 
+func loadGPG(workdir string) error {
+	importpub := exec.Command("gpg", "--no-default-keyring", "--keyring", path.Join(workdir, "pubring.gpg"), "--secret-keyring", path.Join(workdir, "secring.gpg"), "--import")
```
Comment (anthonybishopric) [inline integration/single-node-slug-deploy/check.go]: Why not just create the pubring upfront and save it in this test directory? You can use `util.From` to make it a little easier to locate.

File: integration/single-node-slug-deploy/check.go [hunk 2]
```diff
@@ -173,14 +316,25 @@ func postHelloManifest(dir string) error {
...
-	store, err := intent.LookupStore(intent.Options{})
+	manifestPath, err = signManifest(manifestPath, dir)
 	if err != nil {
 		return err
 	}
-	hostname, _ := os.Hostname()
-	_, err = store.SetPod(hostname, *manifest)
-	return err
+
+	return exec.Command("p2-schedule", manifestPath).Run()
```
Comment (anthonybishopric) [inline integration/single-node-slug-deploy/check.go]: :star: nice expansion of test scope + refactoring

**Comment by anthonybishopric** [comment]
LGTM with the one nit about saving the pubring out of band

File: integration/single-node-slug-deploy/check.go (hunk 1 above)
Comment (tummychow) [inline integration/single-node-slug-deploy/check.go]: good point, I'll add the rings to our repo instead of creating them at test time

**Pull request merged**